src/data/services/LmsApiService.ts

```diff
@@ -3,6 +3,22 @@ import { camelCaseObject } from '@edx/frontend-platform/utils';
 
 import { configuration } from '../../config';
 import generateFormattedStatusUrl from './apiServiceUtils';
+import { EnterpriseGroup, HttpResponse, Paginated } from '../../types';
+
+export type CreateEnterpriseGroupArgs = {
```

Comment:
[curious] I'm interested in your thoughts on when to use `type`  vs. `interface` for object type definitions.

Reply:
IMO, types are for data specifications, while interfaces are for class/function contracts.  In the case of a TypeScript project that is a web app front-end (as opposed to a library or back-end api), I don't see what I consider the ideal use cases for interfaces coming up all that often.
				
Also, I find the operations available to types more generally useful, especially given the heavily composition-oriented (rather than inheritance-oriented/OOP) paradigm coming from our Javascript codebase.